### PR TITLE
[MI-2445] Review fixes

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
1. Added slash command to change the show_prejoin_page setting.

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -69,6 +72,17 @@ func getAutocompleteData() *model.AutocompleteData {
 	embedded.AddStaticListArgument("Choose where the Jitsi meeting should open", true, items)
 	settings.AddCommand(embedded)
 
+	showPrejoinPage := model.NewAutocompleteData("show_prejoin_page", "[value]", "Choose wheather the pre-join page should be visible on Jitsi meeting in embedded mode")
```
Comment (Kshitij-Katiyar) [inline server/command.go]:
```suggestion
	showPrejoinPage := model.NewAutocompleteData("show_prejoin_page", "[value]", "Choose whether the pre-join page should be visible on Jitsi meeting in embedded mode")
```

File: server/command.go [hunk 2]
```diff
@@ -69,6 +72,17 @@ func getAutocompleteData() *model.AutocompleteData {
...
+	items = []model.AutocompleteListItem{{
+		HelpText: "Pre-join page on Jitsi meeting will be displayed",
+		Item:     "true",
+	}, {
+		HelpText: "Pre-join page on Jitsi meeting will be displayed",
+		Item:     "false",
+	}}
+	showPrejoinPage.AddStaticListArgument("Choose wheather the pre-join page should be visible on Jitsi meeting in embedded mode", true, items)
```
Comment (Kshitij-Katiyar) [inline server/command.go]:
```suggestion
	showPrejoinPage.AddStaticListArgument("Choose whether the pre-join page should be visible on Jitsi meeting in embedded mode", true, items)
```

File: server/command.go [hunk 3]
```diff
@@ -69,6 +72,17 @@ func getAutocompleteData() *model.AutocompleteData {
 	embedded.AddStaticListArgument("Choose where the Jitsi meeting should open", true, items)
 	settings.AddCommand(embedded)
 
+	showPrejoinPage := model.NewAutocompleteData("show_prejoin_page", "[value]", "Choose wheather the pre-join page should be visible on Jitsi meeting in embedded mode")
+	items = []model.AutocompleteListItem{{
+		HelpText: "Pre-join page on Jitsi meeting will be displayed",
```
Comment (Kshitij-Katiyar) [inline server/command.go:81]: Here i think its better to have different help text for these two options like 
```suggestion
		HelpText: "Show the pre-join page on Jitsi meeting will be displayed",
```

File: server/command.go [hunk 4]
```diff
@@ -69,6 +72,17 @@ func getAutocompleteData() *model.AutocompleteData {
...
+		Item:     "true",
+	}, {
+		HelpText: "Pre-join page on Jitsi meeting will be displayed",
```
Comment (Kshitij-Katiyar) [inline server/command.go]:
```suggestion
		HelpText: "Don't show the pre-join page on Jitsi meeting will be displayed",
```

File: server/command.go [hunk 5]
```diff
@@ -174,6 +188,7 @@ func (p *Plugin) executeHelpCommand(c *plugin.Context, args *model.CommandArgs)
 
 ###### Jitsi Settings:
 * |/jitsi settings embedded [true/false]|: (Experimental) When true, Jitsi meeting is embedded as a floating window inside Mattermost. When false, Jitsi meeting opens in a new window.
+* |/jitsi settings show_prejoin_page [true/false]|: When false, pre-join page will not be displayed when Jitsi is embedded inside Mattermost.
```
Comment (Kshitij-Katiyar) [inline server/command.go]:
```suggestion
* |/jitsi settings show_prejoin_page [true/false]|: When false, pre-join page will not be displayed when Jitsi meet is embedded inside Mattermost.
```

File: server/command.go [hunk 6]
```diff
@@ -69,6 +72,17 @@ func getAutocompleteData() *model.AutocompleteData {
 	embedded.AddStaticListArgument("Choose where the Jitsi meeting should open", true, items)
 	settings.AddCommand(embedded)
 
+	showPrejoinPage := model.NewAutocompleteData("show_prejoin_page", "[value]", "Choose wheather the pre-join page should be visible on Jitsi meeting in embedded mode")
+	items = []model.AutocompleteListItem{{
+		HelpText: "Pre-join page on Jitsi meeting will be displayed",
+		Item:     "true",
```
Comment (Kshitij-Katiyar) [inline server/command.go]: if you have already created a constant for true and false then use that here

File: server/command.go [hunk 7]
```diff
@@ -289,11 +306,26 @@ func (p *Plugin) executeSettingsCommand(c *plugin.Context, args *model.CommandAr
 			})
 			userConfig = nil
 		}
+	case "show_prejoin_page":
```
Comment (Kshitij-Katiyar) [inline server/command.go]: constant for this

File: server/command_test.go [hunk 8]
```diff
@@ -42,6 +42,7 @@ func TestCommandHelp(t *testing.T) {
 
 ###### Jitsi Settings:
 * |/jitsi settings embedded [true/false]|: (Experimental) When true, Jitsi meeting is embedded as a floating window inside Mattermost. When false, Jitsi meeting opens in a new window.
+* |/jitsi settings show_prejoin_page [true/false]|: When false, pre-join page will not be displayed when Jitsi is embedded inside Mattermost.
```
Comment (Kshitij-Katiyar) [inline server/command_test.go]: same

**Review by Kshitij-Katiyar — APPROVED** [review]

File: server/command.go [hunk 9]
```diff
@@ -58,18 +65,29 @@ func getAutocompleteData() *model.AutocompleteData {
...
 	embedded.AddStaticListArgument("Choose where the Jitsi meeting should open", true, items)
 	settings.AddCommand(embedded)
 
-	namingScheme := model.NewAutocompleteData("naming_scheme", "[value]", "Select how meeting names are generated")
+	showPrejoinPage := model.NewAutocompleteData(commandArgShowPrejoinPage, "[value]", "Choose whether the pre-join page should be visible on Jitsi meeting in embedded mode")
+	items = []model.AutocompleteListItem{{
+		HelpText: "Pre-join page on Jitsi meeting will be displayed",
+		Item:     valueTrue,
+	}, {
+		HelpText: "Pre-join page on Jitsi meeting will be not displayed",
```
Comment (manojmalik20) [inline server/command.go]:
```suggestion
		HelpText: "Pre-join page on Jitsi meeting will not be displayed",
```

File: server/command.go [hunk 10]
```diff
@@ -320,7 +356,7 @@ func (p *Plugin) executeSettingsCommand(c *plugin.Context, args *model.CommandAr
 		Message: p.b.LocalizeWithConfig(l, &i18n.LocalizeConfig{
 			DefaultMessage: &i18n.Message{
 				ID:    "jitsi.command.settings.updated",
-				Other: "Jitsi settings updated",
+				Other: fmt.Sprintf("Jitsi settings updated:\n\n* %s: `%s`", parameters[0], parameters[1]),
```
Comment (manojmalik20) [inline server/command.go:359]:
```suggestion
				Other: fmt.Sprintf("Jitsi settings updated:\n\n %s: `%s`", parameters[0], parameters[1]),
```
What's the need of `*` here? Also, the values you're printing seem wrong.

Comment (raghavaggarwal2308) [reply server/command.go:359]: It's a markdown for bullet mark:
Screenshot from 2022-12-13 15-37-07

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**